app/views/content/is-teaching-right-for-me/career-progression.md

```diff
@@ -0,0 +1,132 @@
+
+To get a TLR you've got to have a proven track record. It’s not just a case of asking for one and you’ll get it. You need to show you’ve got good relationships with your classes. Your teaching has to be consistently solid and you’ve got to get decent results. And that does take time. 
+
+<strong>Elaine:</strong> I was ready for leadership after 6 years of teaching. Because I'd done a master's degree I was leapfrogged. So you can't put years on it. I think you know when you know.
+
+## What are good ways to let people know you’re ready to progress?
+
+<strong>Fran:</strong> If you put yourself forward for opportunities, or you're brave enough to say I don't mind somebody filming my lesson and giving me feedback, it can be a way to make good progress.
+
+<strong>Sian:</strong> Voice to the senior leadership team that career progression is something you're interested in when you have your performance management review. If you don't say anything they'll assume you're happy staying as you are. I had a supportive head that wanted me to do a [national professional qualification (NPQ)](https://www.gov.uk/guidance/national-professional-qualification-npq-courses).
+
+## How can you get more responsibilities?
+
+<strong>Fran:</strong> Realistically you’d want to do at least a year or two in the same setting before then thinking about applying for a role with more responsibility. But you have to prove your worth to get a teaching and learning responsibility (TLR) payment. 
+
+The headteacher at my school was looking to create an alternative for key stage 3 (KS3) pupils who weren’t able to access the curriculum properly. As part of a shift in my lead practitioner responsibility for English I was given a permanent TLR to research and implement this. I had to come up with case studies and a proposal. It's about showing you’re prepared to invest in your personal development and know what’s going on out there. You have to keep up with legislation and news and topical priorities. 
+
+<strong>Sian:</strong> It may be the case that your school hasn’t got the budget for a TLR. So if you really want to take on more responsibility you may need to think about moving school.
+
+## How can teachers increase their salary? 
+
+<strong>Fran:</strong> If you have qualified teacher status (QTS) and work in a maintained school, a school funded by the local authority, your salary is reviewed every year. Most teachers on the main pay scale get an annual pay rise but it may depend on your school’s management performance data to prove you’re completing whole school and individual targets. So you could potentially go from a starting salary of £30,000 to over £41,000, or more if you’re in London, in 5 years.
```

Comment:
```suggestion
<strong>Fran:</strong> If you have qualified teacher status (QTS) and work in a maintained school, a school funded by the local authority, your salary is reviewed every year. Most teachers on the main pay scale get an annual pay rise but it may depend on your school’s performance data to prove you’re completing whole school and individual targets. So you could potentially go from a starting salary of £30,000 to over £41,000, or more if you’re in London, in 5 years.
```

Is `management performance data` the right term? Feels a bit confusing - can we just say `performance`?

Also - it reads like you as an **individual** need to show you are meeting whole school and individual targets - is that right (probably)?

Reply:
I think that must be right as the pay team were OK with it? This is what we say on our pay page:
Your salary will be reviewed every year, with most teachers moving up the pay range annually. This will depend on your school’s performance management arrangements.